### PR TITLE
Build base es-note component

### PR DESCRIPTION
**Pull request by pbishop16**
Addresses #12

### CODE REVIEW

**Comment by pbishop16** [comment]
@MelSumner @rwwagner90 ready for review when you all have a moment. I made some minor additions, please let me know if they need adjustment or are unnecessary.

**Comment by rwwagner90** [comment]
@pbishop16 my only immediate concerns are it would be nice to use flexi, instead of hand rolling the flex styles, but I can work on that conversion later.

Other than that, we probably want to avoid using jQuery, if possible.

**Comment by pbishop16** [comment]
@rwwagner90 Definitely makes sense. Would my removal of the flex styles in this PR help at this time? Also, I will be sure to replace the jQuery usage.

**Comment by rwwagner90** [comment]
@pbishop16 no need to remove the flex styles. We need to refactor everything to use flexi, so you can leave those in until we do that.

If you could remove the jQuery usage, that would be great!

**Comment by pbishop16** [comment]
@rwwagner90 Sounds great. Will do.

**Comment by rwwagner90** [comment]
@pbishop16 I also don't mind cleaning up the jQuery usage later, if you would like me to. Just figured it would be easier to remove before merging in.

**Comment by pbishop16** [comment]
@rwwagner90 Definitely makes sense to remove before merging. Would you mind me using `ember-native-dom-helpers` to replace jQuery usage? I have a commit ready to go, but I thought it best to double check.

**Comment by rwwagner90** [comment]
@pbishop16 that would be great! I love ember-native-dom-helpers

**Comment by pbishop16** [comment]
@rwwagner90 Same here. The corrections have been pushed.

File: addon/constants/mascots.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+export const Mascots = [
+  { image: 'assets/images/icon-tomster.png', name: 'Tomster' },
+  /* Placeholder: */
+  /* { image: 'assets/images/icon-zoey.png', name: 'Zoey' }, */
```
Comment (rwwagner90) [inline addon/constants/mascots.js]: Shouldn't we have both mascots as choices?

Comment (pbishop16) [reply addon/constants/mascots.js]: @rwwagner90 Oh, unfortunately, we do not have an image for Zoey. @MelSumner was working to find an image for 'Zoey', but she wanted me to keep the PR moving in the meantime. I can definitely find a better way to label this.

Comment (rwwagner90) [reply addon/constants/mascots.js]: No worries, if @MelSumner is cool with it being commented out for now, it's definitely fine with me.

**Comment by rwwagner90** [comment]
This LGTM! @MelSumner would you like to take a look before we merge?

**Pull request merged**